front/lib/strapi/appointments/get.ts

```diff
@@ -1,4 +1,7 @@
-import { strapiGet } from '../strapiCRUD';
+import { strapiGet, strapiGetPopulation } from '../strapiCRUD';
 
 export const strapiAppointmentGet = async (filters?: Object) =>
     await strapiGet({ req_url: 'appointments', filters });
+
+export const strapiAppointmentGetPopulation = async (req?: string) =>
```

Comment:
I'd merge it with strapiAppointmentGet by adding to strapiAppointmentGet additional parameter like (filters, population)